lib/Doctrine/DBAL/Migrations/Tools/Console/Command/VersionCommand.php

```diff
@@ -55,9 +55,11 @@ protected function configure()
             ->setName('migrations:version')
             ->setDescription('Manually add and delete migration versions from the version table.')
             ->addArgument('version', InputArgument::OPTIONAL, 'The version to add or delete.', null)
+            ->addArgument('version_end', InputArgument::OPTIONAL, 'The version to add or delete.', null)
```

Comment:
`Command` component set order of options and arguments.
It set options for beginning and arguments after that:

`migrations:version [--add] [--delete] [--all] [--range] [--configuration[="..."]] [--db-configuration[="..."]] [version] [version_end]`